src/Raven.Client/Documents/Indexes/IndexProgress.cs

```diff
@@ -8,6 +8,8 @@ public class IndexProgress
 
         public IndexType Type { get; set; }
 
+        public IndexSourceType SourceType { get; set; }
```

Comment:
like I said I think we don't want to expose that as this is constant property - and should be part of index output not progress. Even if we have race and index type isn't send we simply don't show such item as we don't know about this index. we can have a look at details tomorrow.

Reply:
What about multi map indexes that have both time series / counters / docs?

Reply:
wow, I wasn't even aware we can mix index sources. Then I assume we have to introduce new type of index source: Mixed?

Reply:
At this point we do not support mixing the sources of maps. I will create an issue to throw when something like that happens, till we want to actually support that.

Reply:
We do need to take this into account, because this is very likely going to _be_ a feature.

Reply:
I don't see the problem with adding `SourceType ` here, we already have `public IndexType Type { get; set; }`

Reply:
For now let's stay with current implementation here - introducing the mixed sources will require much more changes in the studio anyway